src/GeoPing.Api/Startup.cs

```diff
@@ -14,37 +14,48 @@
 using GeoPing.Api.Services;
 using GeoPing.Utilities.Logger;
 using IdentityServer4;
+using IdentityServer4.AccessTokenValidation;
+using System.Reflection;
 
 namespace GeoPing.Api
 {
     public class Startup
     {
-        public IConfiguration Configuration { get; }
+        public IHostingEnvironment _environment;
+        public IConfigurationRoot _configuration { get; }
 
-        public Startup(IConfiguration configuration)
+        public Startup(IHostingEnvironment environment)
         {
-            Configuration = configuration;
+            _environment = environment;
+
+
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(environment.ContentRootPath)
+                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+                .AddJsonFile($"appsettings.{environment.EnvironmentName}.json", optional: true);
+
+            builder.AddEnvironmentVariables();
+            _configuration = builder.Build();
```

Comment:
Doesn't it build configuration also using enviroment files by default, should we do i manually?

Reply:
As I know, it builds config using enviroment name and takes unfounded values from appsettings, am I wrong?